src/chips.js

```diff
@@ -49,9 +48,12 @@ const chips = [
       "Maximum current": {
         color: "#FFC869",
         pins: {
-          "sink: ?": ["PA7", "PA6", "PA5", "PA4", "PA3", "PA0", "PB7", "PB6", "PB5", "PB4", "PB3", "PB2", "PB1", "PB0"],
-          "drive: ?": ["PA7", "PA6", "PA5", "PA4", "PA3", "PA0", "PB7", "PB6", "PB5", "PB4", "PB3", "PB2", "PB1", "PB0"]
-          // TODO: I don't understand the datasheet.
+          "sink: 5mA":        [              "PA5"                                                                             ],
+          "sink: 2mA/6mA":    ["PA7", "PA6",                             "PB7",               "PB4", "PB3",        "PB1"       ],
+          "sink: 2mA/10mA":   [                     "PA4", "PA3", "PA0",        "PB6", "PB5",               "PB2",        "PB0"],
+
+          "drive: 0mA":      [              "PA5",                                                                             ],
```

Comment:
Are you sure that PA5 cannot drive current? The datasheet does not seem to explicitly mention that:
![image](https://user-images.githubusercontent.com/2145092/86650603-1e984b00-bfe3-11ea-9241-9d282c457e66.png)

Edit: Ah, it is mentioned here ("open drain output"):

![image](https://user-images.githubusercontent.com/2145092/86650818-50a9ad00-bfe3-11ea-819e-a4246d10893d.png)